generators/generator-base-private.js

```diff
@@ -625,17 +625,16 @@ module.exports = class extends Generator {
     }
 
     /**
-     * Execute callback if git is installed
+     * Checks if git is installed.
      *
-     * @param {function} callback - function to be called if git is installed
+     * @return {boolean} true if installed; false otherwise..
      */
-    isGitInstalled(callback) {
-        this.gitExec('--version', { trace: false }, code => {
-            if (code !== 0) {
-                this.warning('git is not found on your computer.\n', ` Install git: ${chalk.yellow('https://git-scm.com/')}`);
-            }
-            if (callback) callback(code);
-        });
+    isGitInstalled() {
+        const gitInstalled = jhipsterUtils.isGitInstalled();
```

Comment:
Since we are setting this.gitInstalled.
This could be:
`const gitInstalled = this.gitInstalled || jhipsterUtils.isGitInstalled();`

Reply:
Since I added back the callback though we should execute it anyways(with code 0) if we want to keep compatibility, right?

Reply:
Sure